src/main/scala/pureconfig/package.scala

```diff
@@ -76,6 +76,20 @@ package object pureconfig {
   }
 
   /**
+    * Load a configuration of type `Config` from the given `Config`, falling back to the default configuration */
+  def loadConfigWithFallBack[Config](conf: TypesafeConfig)(implicit conv: ConfigConvert[Config]): Try[Config] = {
```

Comment:
I also would like to use the same naming of typesafe config. Can we change `FallBack` to `Fallback`?